lib/transactions.js

```diff
@@ -127,6 +127,40 @@ exports.createGeneration = (blockHeight, blockReward, feeReward, recipients, poo
                     scriptFoundersCompile(foundersAddrHash),
                     Math.round(blockReward * (coin.percentTreasuryReward / 100))
                 )
+            } else if (coin.payAllFounders === true){
```

Comment:
Can you fix the alignment here? I'm also seeing some lines mixing spaces and tabs. We use 4 spaces for each tab.